keymaps/git.cson

```diff
@@ -3,6 +3,19 @@
   'ctrl-shift-8': 'github:toggle-github-tab'
   'ctrl-9': 'github:toggle-git-tab-focus'
   'ctrl-shift-9': 'github:toggle-git-tab'
+  'alt-g enter': 'github:commit'
+  'alt-g f': 'github:fetch'
+  'alt-g F': 'github:pull'
```

Comment:
We've moved away from using the capital letter form because it was confusing for some people that keymaps are case-sensitive. This (and the others like it) should be `alt-g shift-f`.